ztests/suite/zqd/services.sh

```diff
@@ -33,7 +35,7 @@ export AWS_ACCESS_KEY_ID=minioadmin
 export AWS_SECRET_ACCESS_KEY=minioadmin
 export AWS_S3_ENDPOINT=http://localhost:$(cat $portdir/minio)
 
-zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data="$zqdroot" -loglevel=warn &
+zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data="$zqdroot" -loglevel=warn > zqd.log 2>&1 &
```

Comment:
Maybe also drop `loglevel=warn` so we get more data?
```suggestion
zqd listen -l=localhost:0 -portfile="$portdir/zqd" -data="$zqdroot" > zqd.log 2>&1 &
```